graylog2-web-interface/package.json

```diff
@@ -24,7 +24,8 @@
     "lint:changes": "./dev/lintChanges.sh",
     "test": "jest --maxWorkers=50%",
     "test:integration": "jest --maxWorkers=50% --config jest.it.config.js",
-    "check-production-build": "node checkProductionBuild.js"
+    "check-production-build": "node checkProductionBuild.js",
+    "format": "prettier --write src"
```

Comment:
In my opinion it also makes sense to apply the formatting for the `test` and maybe even the `docs` directory.